src/app/route/route-list/route-list.component.css

```diff
@@ -77,6 +77,27 @@
   margin-right: 10px;
 }
 
+.mobile-grid {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 1rem;
+}
+
+.desktop-grid {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-gap: 1rem;
+}
+
+.list-group-item {
```

Comment:
we have another css class named like this (line 8). Delete prev one if necessary